tasks/db_deploy/test/test_db_deploy.py

```diff
@@ -239,33 +253,585 @@ def return_cursor_return_function(conn: connection):
+            call(postgres_con_cursor,
+                 f"{ddl_dir}{db_deploy.SEP}database{db_deploy.SEP}database_comment.sql"),
             call(postgres_con_cursor, f"{ddl_dir}{db_deploy.SEP}roles{db_deploy.SEP}app_role.sql"),
             call(postgres_con_cursor, f"{ddl_dir}{db_deploy.SEP}roles{db_deploy.SEP}appdbo_role.sql"),
             call(postgres_con_cursor, f"{ddl_dir}{db_deploy.SEP}users{db_deploy.SEP}dbo.sql"),
             call(postgres_con_cursor, f"{ddl_dir}{db_deploy.SEP}users{db_deploy.SEP}appuser.sql"),
-            call(another_postgres_con_cursor, f"{ddl_dir}{db_deploy.SEP}schema{db_deploy.SEP}app.sql"),
-            call(another_postgres_con_cursor, f"{ddl_dir}{db_deploy.SEP}tables{db_deploy.SEP}{script_filename_1}"),
-            call(another_postgres_con_cursor, f"{ddl_dir}{db_deploy.SEP}tables{db_deploy.SEP}{script_filename_2}")
+            call(another_postgres_con_cursor, f"{ddl_dir}{db_deploy.SEP}schema{db_deploy.SEP}app.sql")
         ], any_order=False)
-        database.query_no_params.assert_has_calls([
-                call(postgres_con_cursor, f"ALTER USER {db_user} WITH PASSWORD '{db_user_pass}';"),
-                call(postgres_con_cursor, f"ALTER USER dbo WITH PASSWORD '{db_user_pass}';"),
-                call(another_postgres_con_cursor, """SET SESSION AUTHORIZATION dbo;""")
+        mock_query_from_file.assert_has_calls([
+            call(another_postgres_con_cursor,
+                 f"{ddl_dir}{db_deploy.SEP}tables{db_deploy.SEP}{script_filename_1}"),
+            call(another_postgres_con_cursor,
+                 f"{ddl_dir}{db_deploy.SEP}tables{db_deploy.SEP}{script_filename_2}")
+        ], any_order=True)
+        self.assertEqual(10, mock_query_from_file.call_count)
+        mock_query_no_params.assert_has_calls([
+            call(postgres_con_cursor, f"ALTER USER {db_user} WITH PASSWORD '{db_user_pass}';"),
+            call(postgres_con_cursor, f"ALTER USER dbo WITH PASSWORD '{db_user_pass}';"),
+            call(another_postgres_con_cursor, """SET SESSION AUTHORIZATION dbo;"""),
+            call(another_postgres_con_cursor, """SET SESSION AUTHORIZATION dbo;"""),
+            call(another_postgres_con_cursor, """SET SESSION AUTHORIZATION dbo;""")
         ], any_order=False)
+        self.assertEqual(5, mock_query_no_params.call_count)
+
+    @patch('boto3.client')
+    @patch('db_deploy.inner_task')
+    def test_task_pulls_correct_environment_variables(self,
+                                                      inner_task_mock,
+                                                      boto3_client_mock):
+        """
+        Verifies that environment variables are passed to inner_task correctly.
+        """
+        db_user_pass = uuid.uuid4().__str__()
+        db_admin_pass = uuid.uuid4().__str__()
+        db_host = uuid.uuid4().__str__()
+        db_name = uuid.uuid4().__str__()
+        db_user = uuid.uuid4().__str__()
+        db_port = uuid.uuid4().__str__()
```

Comment:
May want to use closer to valid data for port. port should be a number. For PostgreSQL default is 5432 but could use a random int converted to string.

Reply:
Makes sense. Fixed.